packages/blade/src/tokens/theme/__tests__/overrideTheme.test.ts

```diff
@@ -0,0 +1,131 @@
+import overrideTheme from '../overrideTheme';
+import paymentsTheme from '../paymentsTheme';
+import { Theme } from '../theme.d';
+
+const invalidOverridesObjectError = '[Blade:overrideTheme]: The overrides object is not valid';
+const invalidBaseThemeError =
+  '[Blade:overrideTheme]: The base theme provided is not a valid Blade theme';
```

Comment:
Since we're checking for these errors explicitly, they can grow out of sync from the code. Maybe we can create a mapping of error types to error messages and then use the same in test and the module.

Reply:
Yes that's the ideal plan. Maybe I shall create one helper in the succeeding PR.